plugins/chat/assets/javascripts/discourse/components/chat-mention-warnings.js

```diff
@@ -2,21 +2,39 @@ import Component from "@glimmer/component";
 import I18n from "I18n";
 import { htmlSafe } from "@ember/template";
 import { inject as service } from "@ember/service";
+import { computed } from "@ember/object";
 
 export default class ChatMentionWarnings extends Component {
   @service siteSettings;
   @service currentUser;
+  @service chatComposerWarningsTracker;
 
-  get unreachableGroupMentionsCount() {
-    return this.args?.unreachableGroupMentions.length;
+  @computed("chatComposerWarningsTracker.unreachableGroupMentions")
```

Comment:
I might be wrong, but I think you can remove all these `@computed` by making `unreaChableMentions` tracked in the service.